doc/user/content/overview/_index.md

```diff
@@ -10,185 +10,72 @@ menu:
     weight: 5
 ---
 
-Materialize is a **streaming database** powered by [Timely](https://github.com/TimelyDataflow/timely-dataflow#timely-dataflow) and
+Materialize is a **streaming database** purpose-built for low-latency
+applications. Data and software engineering teams use it to solve problems
+where data must be processed at speeds and scales not possible in traditional
+databases, but without the cost, complexity, and development time of most
+streaming engines.
+
+If you're looking to reduce the time it takes to refresh answers to common
+queries, Materialize can make that happen! Rather than recalculating results
+from scratch each time it's queried, or serving stale cached results,
+Materialize continually ingests data and keeps results up to date for you
+as **incrementally updated materialized views**.
+
+<br>
+
+| ![streaming_analytics.gif](https://user-images.githubusercontent.com/23521087/233637537-4c2e5133-5a93-4d9d-9800-a0d4ba842c82.gif) |
```

Comment:
What do you think about using this Metabase dashboard as an example, @jseldess @tr0njavolta?

Reply:
Maybe we can come up with something more...official. This is just a GIF I had lying around! @ruf-io

Reply:
What do you think about instead showing a graphic illustrating "incrementally updated materialized views" - so it'd be more utilitarian, two source tables, a SQL statement and an output, and showing how a change in input drives a quick change in output. Something like what we have on the homepage, but with source data tables added. 
![image](https://user-images.githubusercontent.com/11527560/234016350-ff10d5fb-acdc-473b-afe4-61e912d940ee.png)

I don't like using the BI dashboard UI here because, even though you say "as an example", since it's the overview page and it's the only visual people might incorrectly pigeonhole Materialize as being a thing for real-time analytics dashboards only.